src/libstd/fs/mod.rs

```diff
@@ -1511,7 +1511,7 @@ mod tests {
         assert_eq!(v, b"foobar\0\0\0\0".to_vec());
 
         // Truncate to a smaller length, don't seek, and then write something.
-        // Ensure that the intermediate zeroes are all filled in (we're seeked
+        // Ensure that the intermediate zeroes are all filled in (we're sought
```

Comment:
I don't think this is correct. Probably should be "We have `seek`ed past the end of the file" or something.